packages/next/package.json

```diff
@@ -105,12 +105,26 @@
     "webpack": "4.44.1",
     "webpack-sources": "1.4.3"
   },
+  "optionalDependencies": {
+    "sharp": "0.26.2"
+  },
   "peerDependencies": {
     "react": "^16.6.0 || ^17",
-    "react-dom": "^16.6.0 || ^17"
+    "react-dom": "^16.6.0 || ^17",
+    "node-sass": "^4.0.0",
+    "sass": "^1.3.0",
+    "fibers": ">= 3.1.0"
```

Comment:
I believe ncc would be building all of these dependencies itself though? So I'm not sure we need to reexternalize them.